frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
     width: 220px;
-    
-    background: $dark-gray;
+    border-right: 1px solid #dcdcdc;
+    background-color: rgba(0, 0, 0, 0);
```

Comment:
Ok. But, IMHO Getting all color variables from a file will make it easy for the new developers to use them properly.